cmd/service.go

```diff
@@ -28,45 +32,67 @@ var serviceCmd = &cobra.Command{
 }
 
 var serviceCreateCmd = &cobra.Command{
-	Use:   "create <service_type> [service_name]",
+	Use:   "create <service_type> --plan <plan_name> [service_name]",
 	Short: "Create a new service",
-	Long: `Create a new service from service catalog to deploy on OpenShift.
+	Long: `Create a new service from service catalog using the plan defined and deploy it on OpenShift.
 
-If service name is not provided, service type value will be used.
+If service name is not provided, service type value will be used. The plan to be used must be passed along the service type
+using this convention <service_type>/<plan>. The parameters to configure the service are passed as a list of key=value pairs.
+The list of the parameters and their type is defined according to the plan selected.
 
 A full list of service types that can be deployed are available using: 'odo catalog list services'`,
-	Example: `  # Create new mysql-persistent service from service catalog.
-  odo service create mysql-persistent
+	Example: `  # Create new postgresql service from service catalog using dev plan and name my-postgresql-db.
+  odo service create dh-postgresql-apb my-postgresql-db --plan dev -p postgresql_user=luke -p postgresql_password=secret
 	`,
-	Args: cobra.RangeArgs(1, 2),
+	Args: cobra.RangeArgs(1, 1),
```

Comment:
it should remain `cobra.RangeArgs(1, 1)`
Otherwise, it is not possible to specify service name 
```
▶ odo service create mysql-persistent mymysql --plan deafult
Error: accepts between 1 and 1 arg(s), received 2
....```

Reply:
I missed this, let me fix it real quick